### ISSUE TITLE
scp & ssh& sshfs addon, mc, python2.7

### ISSUE THREAD

**Issue by dzelionis** [icedream/denon-prime4#32]
Some addons....

![image](https://user-images.githubusercontent.com/41810641/197411554-2fa535ae-b826-4474-a2a8-64b41286ecf9.png)

![image](https://user-images.githubusercontent.com/41810641/197412316-fd712914-598d-4485-a1e2-cca38cdbaf83.png)

![image](https://user-images.githubusercontent.com/41810641/197414608-fce8dbde-0fe7-4f88-b26a-e12be64eff29.png)

![image](https://user-images.githubusercontent.com/41810641/197414739-84024b4f-0c78-4c0d-b20d-35d9b850ce74.png)

https://drive.google.com/drive/folders/16OgLBTAjK9EXcM2kH4YsaYXaZdPS66Cc?usp=sharing

**Comment by icedream** [comment]
Interesting to see binaries built using Freetz running on this! Never had that idea, I just used buildroot to take care of this. Pretty cool stuff!

**Comment by dzelionis** [comment]
lol, its very handy for me, as i have few fritzboxes at home and i am running freetz-ng on it...and freetz-ng is build using menuconfig, which includes tones of applications etc.... also its arm cpu so there is no reason why it should not work on denon prime

**Comment by icedream** [comment]
I used freetz-ng for my routers as well but in regards to the Prime 4 was immediately concerned regarding GPU and UI dependencies that probably would not have been shipped with it. At that point I also already knew that Buildroot was used to compile the official firmware anyways.

Buildroot also has a similar menu to build packages. You'll notice I added a `buildroot-customizations/` directory to provide my own packages to the system, I can later just put checkmarks in that menu for the packages I want to build and it handles dependencies and everything as well from there.

I just wrote scripts to check out the correct Buildroot version and configure and compile it:

- [clone-buildroot.sh](https://github.com/icedream/denon-prime4/blob/icedream-experiments/clone-buildroot.sh) extracts the Buildroot version to be used from the firmware and checks it out with git
- [configure-buildroot.sh](https://github.com/icedream/denon-prime4/blob/icedream-experiments/configure-buildroot.sh) runs the configuration tool in Buildroot and copies its output back into my own Git file tree.
- [compile-buildroot.sh](https://github.com/icedream/denon-prime4/blob/icedream-experiments/compile-buildroot.sh) does the building and then filters the output for files to be copied onto the firmware rootfs (the biggest hack of these three lol).

Maybe something for someone else to experiment with as well :D
